examples/ui/client-capabilities/interaction/text-area.js

```diff
@@ -0,0 +1,8 @@
+import { CodeLabel, CodeSection } from '../../documentationBase';
+import { registerComponents } from '../../rapidImports';
+
+registerComponents();
+
+export default function TextAreaDemo({ children, color }) {
+	return (<CodeSection><rapid-text-area maxlength="20" placeholder="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua" rows="10" cols="10">Text Area</rapid-text-area></CodeSection >)
```

Comment:
I meant several different text areas with different configs

Reply:
I am not sure if its worth adding many different examples, the href link at the top links to the underlying component which has more examples:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea
Im not even sure it's worth adding the attributes that are there as they are all just duplicated out from the underlying component. Aren't we just duplicating information and not adding anything new?

Reply:
I expect a lot of developers using these docs won't be front end specialists and know the difference. It's useful to show the different ways a developer can use it.

Reply:
I've added two examples now